conftest.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import os
+from typing import Any
+
+import pytest
+import requests
+
+import openeo
+from openeo.capabilities import ComparableVersion
+
+_log = logging.getLogger(__name__)
+
+def get_openeo_base_url(version: str = "1.1.0"):
+    try:
+        endpoint = os.environ["ENDPOINT"].rstrip("/")
+    except Exception:
+        raise RuntimeError("Environment variable 'ENDPOINT' should be set"
+                           " with URL pointing to OpenEO backend to test against"
+                           " (e.g. 'http://localhost:8080/' or 'http://openeo-dev.vgt.vito.be/')")
+    return "{e}/openeo/{v}".format(e=endpoint.rstrip("/"), v=version)
+
+
+def get_tsservice_base_url():
```

Comment:
I guess this "tsservice" stuff is not used in the benchmarks? so I would not drag in these things here